### DISCUSSION TITLE
New Car

### DISCUSSION THREAD

**Discussion by commanderi1972** [teslamate-org/teslamate#3994] [Q&A]
I have had a new Model Y since Saturday and now we are no longer shown consumption figures for the new car under drives
![IMG_8631](https://github.com/teslamate-org/teslamate/assets/157980603/c64f222a-1f00-4d0a-ba62-80a4b46710b4)

**Comment by cwanja** [comment]
Have you completed the charges outlined in the FAQs? https://docs.teslamate.org/docs/faq#why-are-no-consumption-values-displayed-in-grafana

**Reply by commanderi1972** [reply]
OK, I only charged it once from 10 to 60 percent. So I have to charge it several times before the values ​​are displayed. Thank you
